wdfn-server/waterdata/templates/base_network.html

```diff
@@ -2,6 +2,9 @@
 
 {% block page_css %}
       <link rel="stylesheet" href="{{ 'network.css' | asset_url }}">
+    {% if request.user_agent.browser == 'msie' %}
```

Comment:
I think it would be more appropriate to put this in the page_script block. If the browser is msie import wdfnviz.js, otherwise import the other script.

Reply:
Oops. Yes, much better choice. Changed.